README.md

```diff
@@ -2,8 +2,10 @@
 A Simplenote client for iOS. Learn more about Simplenote at [Simplenote.com](https://simplenote.com).
 
 ## Development Requirements
-* A Simperium account. [Sign up here](https://simperium.com/signup/)
-* A Simperium Application ID and key. [Create a new app here](https://simperium.com/app/new/)
+* A Simperium account ([sign up here](https://simperium.com/signup/)).
+* A Simperium Application ID and key ([create a new app here](https://simperium.com/app/new/)).
```

Comment:
@@ -2,8 +2,10 @@
 A Simplenote client for iOS. Learn more about Simplenote at [Simplenote.com](https://simplenote.com).
 
+* A Simperium account ([sign up here](https://simperium.com/signup/)).
+* A Simperium Application ID and key ([create a new app here](https://simperium.com/app/new/)).
+* [CocoaPods](https://cocoapods.org/).
+* [Xcode](https://developer.apple.com/xcode/).
 
 ## Running